src/d3a/d3a_core/export.py

```diff
@@ -517,6 +607,26 @@ def plot_bar_graph(cls, barmode: str, title: str, xtitle: str, ytitle: str, data
         fig = go.Figure(data=data, layout=layout)
         py.offline.plot(fig, filename=iname, auto_open=False)
 
+    @classmethod
+    def plot_line_graph(cls, title: str, xtitle: str, ytitle: str, data, iname: str, xmax: int):
```

Comment:
It is quite bad that there is a function called 'plot_line_graph' inside a class called 'BarGraph'. I would convert the plot_line_graph to a free function, since there is not really any dependency on the class members (it is a classmethod anyways, and cls is not used).